sql/rpl_parallel.cc

```diff
@@ -918,6 +922,7 @@ retry_event_group(rpl_group_info *rgi, rpl_parallel_thread *rpt,
       err= rgi->worker_error= 1;
       my_error(ER_PRIOR_COMMIT_FAILED, MYF(0));
       mysql_mutex_unlock(&entry->LOCK_parallel_entry);
+
```

Comment:
Please avoid unrelated formatting changes to the existing code, as they complicate merges and history tracking.